.gitignore

```diff
@@ -8,5 +8,4 @@ tmp/
 *.jld2
 *.cov
 *.info
-/test/Manifest.toml
-/Manifest.toml
+Manifest.toml
```

Comment:
Seems you forgot to remove `docs/Manifest.toml` in this PR? Or was it kept on purpose?

Reply:
Oh right I must have forgotten to remove it